modules/extensions/examples/creating-a-telemetry-plugin-for-devworkspaces/che_cluster_with_custom_plugin.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  creationTimestamp: '2020-05-14T13:21:51Z'
+  finalizers:
+    - oauthclients.finalizers.che.eclipse.org
+  generation: 18
+  name: {prod-checluster}
+  namespace: __<{prod-namespace}>__
+  resourceVersion: '5108404'
+  selfLink: /apis/org.eclipse.che/v1/namespaces/che/checlusters/eclipse-che
+  uid: bae08db2-104d-4e44-a001-c9affc07528d
+spec:
+  auth:
+    identityProviderURL: 'https://keycloak-che.apps-crc.testing'
+    identityProviderRealm: che
+    updateAdminPassword: false
+    oAuthSecret: ZMmNPRbgOJJQ
```

Comment:
not sure if we need to provide detailed CR after the creation with values like `oAuthSecret` 
I think it might make the docs less straight-forward

Reply:
Oops, good catch, this file is actually not used in this PR